Shared/Theme/Themeable.swift

```diff
@@ -21,11 +22,19 @@ extension Themeable {
         themeObserver = notificationCenter.addObserver(name: .ThemeDidChange,
                                                        queue: mainQueue) { [weak self] _ in
```

Comment:
I can't put comment on the `public func listenForThemeChange() {` (without view parameter) but that one is now useless I believe since only the function with the parameter is called, so it could be removed?

Reply:
@lmarceau did you see my comment?

Reply:
This aligns and gives me confidence in getting rid of it completely

Reply:
@nbhasin2 I had not haha, I had started the review and saw your comment afterwards. Yes we align, IMO good to remove!